float32 center_x
float32 center_y
float32 width
float32 height
float32 confidence
float32 class_id